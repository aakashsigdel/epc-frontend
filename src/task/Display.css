.display-container {
  width: 200px;
  margin: 5px auto;
  padding: 15px;
  border: 1px solid #5f93a7;
}

.display-container div {
  background-color: rgba(128, 128, 128, 0.3);
  overflow: scroll;
  color: white;
  height: 18px;
}
