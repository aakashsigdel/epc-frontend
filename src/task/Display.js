import React from 'react';

const Display = props =>
  <div></div>;

export default Display;
