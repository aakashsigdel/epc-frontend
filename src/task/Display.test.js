import React from 'react';
import Display from './Display';

import renderer from 'react-test-renderer';

it('should render <Display /> correctly', () => {
  const component = renderer.create(
    <Display
      inputText={'hello'}
      bgColor={'#fff'}
      onClick={() => null}
    />
  );
  let snapshotTree = component.toJSON();
  expect(snapshotTree).toMatchSnapshot();

  snapshotTree.props.onClick();
  snapshotTree = component.toJSON();
  expect(snapshotTree).toMatchSnapshot();
});

