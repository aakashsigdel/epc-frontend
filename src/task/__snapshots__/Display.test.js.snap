// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <Display /> correctly 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#fff",
      "height": 100,
      "width": 400,
    }
  }
>
  <span
    style={
      Object {
        "backgroundColor": "grey",
        "opacity": 0.8,
      }
    }
  >
    hello
  </span>
</div>
`;

exports[`should render <Display /> correctly 2`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#fff",
      "height": 100,
      "width": 400,
    }
  }
>
  <span
    style={
      Object {
        "backgroundColor": "grey",
        "opacity": 0.8,
      }
    }
  >
    hello
  </span>
</div>
`;
